examples/system_example/Kconfig

```diff
@@ -0,0 +1,16 @@
+menu "m4a-firmware configuration"
```

Comment:
It's better to these cases call rsource/m4a-firmware/Kconfig. because could be generate copies of the sames CONFIG_PARAMS